src/test/java/ActionsTests.java

```diff
@@ -1,117 +1,93 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
-import org.testng.asserts.SoftAssert;
+import pageObject.AllSongsPage;
+import pageObject.HomePage;
+import pageObject.LoginPage;
 
 import java.util.List;
 
 public class ActionsTests extends BaseTest {
 
-    @Test
-    public void playSongTest() {
-        // hover over in clickPlayBtn
-        enterEmail("demo@class.com");
-        enterPassword("te$t$tudent");
-        loginSubmit();
-        clickPlayBtn();
-        Assert.assertTrue(pauseBtnExists());
-
-        // Comparing numbers of elements example
-        List<WebElement> songs = driver.findElements(By.cssSelector("[data-test='song-card']"));
-
-        int songsNumberBefore = songs.size();
-        System.out.println(songsNumberBefore);
-        // Just an example: here we would add or delete an element but we didn't
-        int songsNumberAfter = songs.size();
-        System.out.println(songsNumberAfter);
-
-        // Soft assert example
-        SoftAssert softAssert = new SoftAssert();
-        softAssert.assertEquals(driver.getCurrentUrl(), "https://bbb.testpro.io/#!/queue");
-        softAssert.assertTrue(songsNumberBefore == songsNumberAfter,
-                "=== Songs number before should be equal songs number after ===");
-        softAssert.assertAll();
-    }
+//    @Test
+//    public void playSongTest() {
+//        // hover over in clickPlayBtn
+//        LoginPage loginPage = new LoginPage(getDriver());
+//        HomePage homePage = new HomePage(getDriver());
+//        AllSongsPage allSongsPage = new AllSongsPage(getDriver());
+//        loginPage.provideEmail("burkovads@mail.ru")
+//                .providePassword("Julka@0721")
+//                .clickSubmitBtn();
+//        homePage.goToAllSongs();
+//        allSongsPage.allShuffle();
+//        Assert.assertTrue(allSongsPage.barIsDisplayed());
+
+//        // Comparing numbers of elements example
+//        List<WebElement> songs = driver.findElements(By.cssSelector("[data-test='song-card']"));
+//
+//        int songsNumberBefore = songs.size();
+//        System.out.println(songsNumberBefore);
+//        // Just an example: here we would add or delete an element, but we didn't
+//        int songsNumberAfter = songs.size();
+//        System.out.println(songsNumberAfter);
+//
+//        // Soft assert example
+//        SoftAssert softAssert = new SoftAssert();
+//        softAssert.assertEquals(driver.getCurrentUrl(), "https://bbb.testpro.io/#!/queue");
+//        softAssert.assertTrue(songsNumberBefore == songsNumberAfter,
+//                "=== Songs number before should be equal songs number after ===");
+//        softAssert.assertAll();
+//    }
+
+//    @Test
+//    public void createPlaylist() {
+//        String createdPlaylistName = "Spring";
+//        LoginPage loginPage = new LoginPage(driver);
+//        HomePage homePage = new HomePage(driver);
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.